### PR TITLE
Hometask2

### PR DESCRIPTION
**Pull request by codelyne**
Description
---------------
Change the project structure
Add webpack, babel, react packages

Demo
--------
http://reactjs.umar0v.name/

### CODE REVIEW

**Comment by codelyne** [comment]
I found one mistake in a webpack.config.js file. In property devtool, it must be false instead of null. Could someone tell me the proper way how to fix it in this pull request?

File: hometask1/hometask1/js/classes.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+class Human {
+  /**
```
Comment (ubcent) [inline hometask1/hometask1/js/classes.js]: Why this changes in the current pull request?

**Comment by ubcent** [comment]
> I found one mistake in a webpack.config.js file. In property devtool, it must be false instead of null. Could someone tell me the proper way how to fix it in this pull request?

You should fix it according to this [doc](https://webpack.js.org/configuration/devtool/)

File: hometask1/hometask1/js/classes.js (hunk 1 above)
Comment (codelyne) [reply hometask1/hometask1/js/classes.js]: In the beginning, I tried to apply new knowledge to hometask1, but something went wrong and I decided not to touch this task =) Just don't care about it.

**Comment by codelyne** [comment]
I didn't use any boilerplate. I just read [this article](https://stanko.github.io/setting-up-webpack-babel-and-react-from-scratch/) and apply that to my home task.

**Comment by codelyne** [comment]
> I found one mistake in a webpack.config.js file. In property devtool, it must be false instead of null. Could someone tell me the proper way how to fix it in this pull request?
>> You should fix it according to this doc

I wanted to know how should I change my pull request? Do I need to make another pull request? The problem was in `null` parameter and I solved it using `false` instead.

**Comment by ubcent** [comment]
You just need to commit this fix to a source branch then your pull request is updating automatically.

File: hometask1/hometask1/js/classes.js (hunk 1 above)
Comment (ubcent) [reply hometask1/hometask1/js/classes.js]: It can influence me because of noises in a code. Could you please avoid it in the future PRs?
